unit_tests/server/parameters_parser.cpp

```diff
@@ -321,6 +325,37 @@ BOOST_AUTO_TEST_CASE(valid_route_urls)
     auto result_13 = parseParameters<RouteParameters>("1,2;3,4");
     BOOST_CHECK(result_13);
     BOOST_CHECK_EQUAL(result_13->generate_hints, true);
+
+    // parse single annotations value correctly
+    RouteParameters reference_14{};
+    reference_14.annotations_type = RouteParameters::AnnotationsType::Duration;
+    reference_14.coordinates = coords_1;
+    auto result_14 = parseParameters<RouteParameters>("1,2;3,4?geometries=polyline&"
+                                                      "overview=simplified&annotations=duration");
+    BOOST_CHECK(result_14);
+    BOOST_CHECK_EQUAL(reference_14.geometries, result_14->geometries);
+    BOOST_CHECK_EQUAL(
+        static_cast<bool>(result_2->annotations_type & RouteParameters::AnnotationsType::Duration),
+        true);
+    BOOST_CHECK_EQUAL(result_14->annotations, true);
+
+    // parse multiple annotations correctly
+    RouteParameters reference_15{};
+    reference_15.annotations_type = RouteParameters::AnnotationsType::Duration |
+                                    RouteParameters::AnnotationsType::Weight |
+                                    RouteParameters::AnnotationsType::Nodes;
+    reference_15.coordinates = coords_1;
+    auto result_15 =
+        parseParameters<RouteParameters>("1,2;3,4?geometries=polyline&"
+                                         "overview=simplified&annotations=duration,weight,nodes");
+    BOOST_CHECK(result_15);
+    BOOST_CHECK_EQUAL(reference_15.geometries, result_15->geometries);
+    BOOST_CHECK_EQUAL(
+        static_cast<bool>(result_2->annotations_type & (RouteParameters::AnnotationsType::Duration |
+                                                        RouteParameters::AnnotationsType::Weight |
+                                                        RouteParameters::AnnotationsType::Nodes)),
+        true);
+    BOOST_CHECK_EQUAL(result_15->annotations, true);
```

Comment:
Order should not matter - care to create a test expressing this?
Also edge cases such as
- empty list
- all possibilities
- all possibilities including All, None and combinations